src/routes/[...lang]/__layout.svelte

```diff
@@ -5,23 +5,35 @@
   import { page } from '$app/stores';
   registerAll();
   let defaultLang = 'en';
-  let localeRegex = new RegExp(`^/(${availableLocales.join('|')})/`)
+  let localeRegex = new RegExp(`^/(${availableLocales.join('|')})(/|$)`)
```

Comment:
I think you have to rebase this line, I did a quickfix for this in which I made the trailing `/` optional, but it should only be optional if it's the end of the path.

Reply:
fixed